adapter/src/dbt/adapters/fal/teleport_support/duckdb.py

```diff
@@ -23,17 +25,35 @@ def teleport_from_external_storage(self, relation: BaseRelation, relation_path:
         url = teleport_info.build_url(relation_path)
 
         with self._db_adapter.connection_named('teleport:copy_from'):
-            rendered_macro = self._db_adapter.execute_macro('duckdb__copy_from', kwargs={'relation': relation, 'url': url})
+            if self.credentials.type == TeleportTypeEnum.REMOTE_S3:
+                # Putting this in __init__ didn't work, looks like it has to be done with each new connection
```

Comment:
That is true! It has to be setup on each

Reply:
Btw, does that mean we should be doing `LOAD 'parquet'` on each connection?

Reply:
For me it failed before. I will test locally.

Reply:
```
❯ duckdb ~/duck_db_dbt_dump.db
v0.5.1 7c111322d
Enter ".help" for usage hints.
D load httpfs;
D SET s3_region='x';
D SET s3_access_key_id='x';
D SET s3_secret_access_key='x';
D SELECT * FROM read_parquet('s3://mybucket/teleport/main.main.orders_forecast.parquet');
Error: Catalog Error: Function with name read_parquet is not on the catalog, but it exists in the parquet extension. To Install and Load the extension, run: INSTALL parquet; LOAD parquet;
```

Reply:
But not putting it the code it still works. I am assuming it is loaded by dbt-duckdb somewhere... But can we still include it to be safe?